oclif/core
# [POC] feat: show aliases under their own help section

## What does this PR do?
This makes aliases show up under their own section in the help output for a CLI instead of having them show up under the COMMANDS.

![image](https://user-images.githubusercontent.com/1084688/164083594-ebf616e1-ee8d-40eb-a133-cce65699115a.png)

## Testing instructions
1. Get this branch locally
    1. Run `yarn install`
    1. Run `yarn link`
1. Clone `https://github.com/RodEsp/test-cli` locally
    1. Run `yarn install`
    1. Run `yarn link @oclif/core`
    1. Run `bin/dev help`
1. See `hello-alias` show up under `ALIASES` instead of `COMMANDS`
